op-challenger/game/fault/agent.go

```diff
@@ -18,6 +19,8 @@ import (
 type Responder interface {
 	CallResolve(ctx context.Context) (gameTypes.GameStatus, error)
 	Resolve(ctx context.Context) error
+	CallResolveClaim(ctx context.Context, claimIdx int64) error
+	ResolveClaim(ctx context.Context, claimIdx int64) error
```

Comment:
Indices should be unsigned `u256` numbers onchain, can we make these `uint64` values?